client/src/js/samples/components/LabelEditor/LabelEditor.js

```diff
@@ -0,0 +1,192 @@
+import React from "react";
+import { connect } from "react-redux";
+import styled from "styled-components";
+import { get, map } from "lodash-es";
+import { Table, ViewHeader, ViewHeaderTitle, RemoveModal, Button, BoxGroup, BoxGroupHeader, Flex } from "../../../base";
+import { getLabels, createLabel, removeLabel, updateLabel } from "../../actions";
+import { CreateLabel } from "./CreateLabel";
+import { LabelItem } from "./LabelItem";
+import { EditLabel } from "./EditLabel";
+
+export const StyledTable = styled(Table)`
```

Comment:
Only use `table` elements for actual tables. You can solve this layout problem better with [flex box](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).